src/views/CheckoutSelectAccount.vue

```diff
@@ -26,7 +26,7 @@ import {RequestType} from '../lib/RequestTypes';
 import {State, Mutation} from 'vuex-class';
 
 @Component({components: {AccountSelector, LoginSelector}})
-export default class Checkout extends Vue {
+export default class CheckoutSelectAccount extends Vue {
```

Comment:
Only a name refactor with no external dependencies, because it's `export default`.